cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -811,12 +811,28 @@ private boolean getState(HttpServletRequest request, HttpServletResponse respons
           out.write(String.format("%50s, %s, %s%n", goal.getClass().getSimpleName(), goal.clusterModelCompletenessRequirements(),
               entry.getValue() ? "Ready" : "NotReady").getBytes(StandardCharsets.UTF_8));
         }
-        if (state.executorState().state() == ExecutorState.State.REPLICA_MOVEMENT_TASK_IN_PROGRESS) {
+        ExecutorState.State executorState = state.executorState().state();
+        if (executorState == ExecutorState.State.REPLICA_MOVEMENT_TASK_IN_PROGRESS
+            || executorState == ExecutorState.State.STOPPING) {
           out.write(String.format("%n%nIn progress partition movements:%n").getBytes(StandardCharsets.UTF_8));
           for (ExecutionTask task : state.executorState().inProgressPartitionMovements()) {
             out.write(String.format("%s%n", task).getBytes(StandardCharsets.UTF_8));
           }
-          out.write(String.format("%n%nPending partition movements:%n").getBytes(StandardCharsets.UTF_8));
+          out.write(String.format("%n%nAborting partition movements:%n").getBytes(StandardCharsets.UTF_8));
```

Comment:
In this case and for the cases of other task states, should we avoid printing this line if there are no tasks in this state?

Reply:
For completeness, we may still want to print it.